sphinx/themes/basic/static/searchtools.js

```diff
@@ -111,12 +111,14 @@ const _finishSearch = (resultCount) => {
   Search.stopPulse();
   Search.title.innerText = _("Search Results");
   if (!resultCount)
-    Search.status.innerText = Documentation.gettext(
+    Search.status.innerText = _(
```

Comment:
Is this change necessary?

It's nitpicky, but I'd generally both to avoid modifying unnecessary code lines, and also to maintain consistency (`Documentation.gettext` alongside `Document.ngettext`, compared to usage of underscore abbreviation in one but not the other.  I admit that the abbreviated forms are often useful).

Reply:
It's not necessary, happy to revert if you like, or make a separate PR with just this change?

I initially changed it because the `_` alias is used everywhere else in this file. That was before adding `Documentation.ngettext`, so I can see the case for symmetry with `Documentation.gettext`.

Personally I would use `_` where possible, so there aren't three forms floating around (`_`, `Documentation.gettext`, `Documentation.ngettext`) but no strong opinion.

Reply:
It does probably seem pedantic, but yep, I think I'd prefer undoing this single-line change.

And then as a follow-up, I'd suggest adding a quick-alias in `doctools.js` for `__` (to complement the existing `_` alias), and migrating both callsites to use that.

Reply:
(the latter I suppose should be considered a feature, despite being a small one)